transformSchema.json

```diff
@@ -1,585 +1,606 @@
+        "type": "object",
+        "required": [
+          "args"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "currentTime"
+            ]
+          },
+          "args": {
+            "type": "object",
+            "required": [
+              "format"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "format": {
+                "description": "The format to parse the time string"
+              }
+            }
+          }
+        }
+      },
+      "inverse": {
+        "description": "Accepts boolean as input, returns inverse boolean",
+        "type": "object",
+        "required": [
+          "type"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "inverse"
+            ]
+          }
+        }
+      },
+      "split": {
+        "description": "Accepts a string and returns an array",
+        "type": "object",
+        "required": [
+          "type",
+          "args"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "split"
+            ]
+          },
+          "args": {
+            "type": "object",
+            "required": [
+              "on"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "on": {
+                "description": "The string to split on",
+                "type": "string"
+              }
+            }
+          }
+        }
+      },
+      "replace": {
+        "description": "Accepts a string, returns a string",
+        "type": "object",
+        "required": [
+          "type",
+          "args"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "replace"
+            ]
+          },
+          "args": {
+            "type": "object",
+            "required": [
+              "regex",
+              "new"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "regex": {
+                "description": "Regex string that must have 1 capture group that will be used to match the part of the string that will be replaced",
+                "type": "string"
+              },
+              "new": {
+                "description": "The value that will replace capture group 1 in the above regex",
+                "type": "string"
+              }
+            }
+          }
+        }
+      },
+      "max": {
+        "description": "Accepts an array and finds the max valuye of these items. Can return a generic or a complex object",
```

Comment:
Extra `y` in `valuye`

Reply:
👍 Might as well fix that too while we're here.